sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -600,6 +600,20 @@ namespace pi2ur {
 
 inline pi_result piTearDown(void *PluginParameter) {
   std::ignore = PluginParameter;
+  // Fetch the 1 known adapter so we can release it. Fetching it for a second

```

Comment:
```suggestion
  // Fetch the first known adapter so we can release it. Fetching it for a second

```

Reply:
I've clarified the comment here, it's actually referring to a single adapter (the one that's statically linked) rather than the first adapter. The PI plugin for unified runtime that supports multiple adapters through the UR loader uses a different piTearDown implementation